R/db.R

```diff
@@ -0,0 +1,15 @@
+get_report_version <- function(db, name, id) {
```

Comment:
this will end up being re-used by a number of endpoints. `api.R` is getting to be a big file; it may be worth putting more of the database querying logic in this file as we keep adding endpoints.